### DISCUSSION TITLE
squeezer has become unusably slow with latest LMS (9.0.0 and prev release)

### DISCUSSION THREAD

**Discussion by pooh22** [LMS-Community/slimserver#1261] [General]
I've been using squeezer and LMS for ages and since recently the android app squeezer is so slow it doesn't make sense to use it anymore. Has anything changed with the API or the server responsiveness to the API?

Interestingly the integration in home assistant for the LMS is much better and I now use it mostly, but that cannot be used to configure playlists and more complicated things than on/off and volume stuff.

**Comment by pssc** [comment]
@pooh22 the HA stuff as got better as the HA squeezebox integration team having put in loads of work, its using the http API via json requests, I am not sure what squeezer uses but i think have it somewhere but mostly just use the material interface directly. We will look at playlists etc but we are limited by the interfaces presented by HA

**Comment by michaelherger** [comment]
I don't think the API should have become considerably slower. And even the new additions which _might_ add some overhead are optional, new tags, which the caller would have to request specifically.

**Comment by pssc** [comment]
@pooh22 have checked out squeezer on my setup seems ok.  So you might have something else going on

**Comment by pooh22** [comment]
thanks for the feedback, I'll check out other issues I might have that could cause slowness, it just coincided with the subsequent installation of the last 8 release and 9.0.0, that I figured it must have been something on the server side...

**Comment by pooh22** [comment]
I now have a new phone, it seems to make a lot of difference to the usability/performance, so while there may be something going on, it's probable this has more to do with squeezer than LMS

**Comment by Paddy4711** [comment]
@pooh22 I had the same issue squeezer was totally unusable. Uninstalling squeezer and installing squeezer again, solved the issue for me. Everything is fast as normal !
